lib/util/ImpCentralApiHelper.js

```diff
@@ -52,6 +53,36 @@ class ImpCentralApiHelper {
         this._authConfig = AuthConfig.getEntity(authLocation, accountId);
         this._impCentralApi = new ImpCentralApi(endpoint || this._authConfig.endpoint);
         this._entityStorage = new EntityStorage();
+        this._setupProxyAgent();
+    }
+
+    _setupProxyAgent() {
+        let httpProxy = process.env.GLOBAL_AGENT_HTTP_PROXY || process.env.HTTP_PROXY || process.env.http_proxy;
+        let httpsProxy = process.env.GLOBAL_AGENT_HTTPS_PROXY || process.env.HTTPS_PROXY || process.env.https_proxy;
+        const majorNodejsVersion = parseInt(process.version.slice(1).split('.')[0], 10);
+        // global-agent module works for NodeJS v10+ only
+        if (majorNodejsVersion >= 10 && (httpProxy || httpsProxy)) {
```

Comment:
Should we throw in an `else if (majorNodejsVersion < 10  && (httpProxy || httpsProxy))` or at least log a warning that log streaming is probably going to fail and to upgrade your node.js version?

Reply:
Thanks, I agree, it should not be silent.
Warning seems to be a better choice, it is added for the commands that use log streams.